packages/fxa-auth-server/lib/routes/subscriptions/stripe.ts

```diff
@@ -561,6 +584,29 @@ export class StripeHandler {
       customer.id,
       paymentMethodId
     );
+
+    if (
+      paymentMethod?.billing_details?.address?.postal_code &&
+      paymentMethodCountry
+    ) {
+      this.stripeHelper.setCustomerLocation({
+        customerId: customer.id,
+        postalCode: paymentMethod.billing_details.address.postal_code,
+        country: paymentMethodCountry,
+      });
+    } else {
+      Sentry.withScope((scope) => {
+        scope.setContext('updateDefaultPaymentMethod', {
+          customerId: customer!.id,
+          paymentMethodId: paymentMethod?.id,
+        });
+        Sentry.captureMessage(
+          `Cannot find a postal code or country for customer ${customer!.id}`,
```

Comment:
Same nit for and/or

Reply:
I was going to make this change along with the other changes.  But since I won't be making the other changes I thought I needed, I'm not going to make this small one.